client/blocks/image-editor/index.jsx

```diff
@@ -21,7 +21,7 @@ import ImageEditorCanvas from './image-editor-canvas';
 import ImageEditorToolbar from './image-editor-toolbar';
 import ImageEditorButtons from './image-editor-buttons';
 import MediaUtils from 'lib/media/utils';
-import closeOnEsc from 'lib/mixins/close-on-esc';
+import closeOnEscHoc from 'lib/mixins/close-on-esc/hoc';
```

Comment:
Absolutely agree. This wasn't meant as a finished piece though, rather just a draft to start a conversation about the approach. While developing I wanted to keep things together and used original-path/hoc for convenience really. I intend to use `lib/close-on-esc` once I start cleaning this PR up in to something that could be merged.